specifyweb/frontend/js_src/lib/schemaextras.ts

```diff
@@ -227,8 +295,19 @@ export const schemaExtras: {
     isOnLoan.isHidden = true;
     isOnLoan.overrides.isHidden = true;
 
+    const preparation = new LiteralField(model, {
```

Comment:
Where is this Loan.preparation field coming from? It is not one of the calculated fields that was added?

Reply:
It's not Loan.preparation. It's Preparation.preparation (the diff is collapsed)

Looks like a copy-paste bug. Fixed in [9d52d4f](https://github.com/specify/specify7/commit/9d52d4f041cea0dc6990740a1ed95a6d548f8296)

Reply:
Yeah sorry. I just realized that and edited the comment. But there should be a totalCountAmt field.

Reply:
The types look correct. All integers.

Reply:
Fixed in [60bad96](https://github.com/specify/specify7/commit/60bad96ea8d3a0658ea43171f009b3a0d2a69b84)